src/oscclient/oscclientmanager.cpp

```diff
@@ -0,0 +1,44 @@
+#include <QMutex>
+#include <QDir>
+#include <QtDebug>
+
+#include "oscclient/oscclientmanager.h"
+#include "engine/sidechain/enginesidechain.h"
+#include "engine/sidechain/engineoscclient.h"
+#include "engine/enginemaster.h"
+
+
+OscClientManager::OscClientManager(UserSettingsPointer& pConfig, EngineMaster* pEngine):
+    timer(this)
+         {
+
+    //Register EngineRecord with the engine sidechain.
+    EngineSideChain* pSidechain = pEngine->getSideChain();
+    if (pSidechain) {
+        EngineOscClient* pEngineRecord = new EngineOscClient(pConfig);
+        pSidechain->addSideChainWorker(pEngineRecord);
+    }
+
+
+}
+
+OscClientManager::~OscClientManager()
+{
+    qDebug() << "Delete OscClient";
+}
+
+
+void OscClientManager::sendState(){
+
+}
+
+
+void OscClientManager::connectServer()
+{
+
+}
+
+void OscClientManager::maybeSendState()
+{
+
+}
```

Comment:
The formatting of this file is not correct. Please have a look here: 
https://www.mixxx.org/wiki/doku.php/coding_guidelines

Reply:
I notice there are several formatting issues with the OSC-related source files. Do I need to fix these manually or is it appropriate to simply run clang-format on all the .cpp/.h files in this pull request?